handler.py

```diff
@@ -114,13 +114,20 @@ def menus_transaction(crawled_meals, cursor):
     new_menus, deleted_menus, edited_menus = compare_menus(db_menus, crawled_meals, restaurants)
 
     if deleted_menus:
-        deleted_menus_id = [str(menu.get("id")) for menu in deleted_menus]
-        delete_menus_query = f"""
-            DELETE FROM menu
-            WHERE id in ({','.join(deleted_menus_id)});
-        """
-        cursor.execute(delete_menus_query)
-    send_deleted_menus_message(deleted_menus)
+        deleted_menus_id = []
+        for menu in deleted_menus:
+            restaurant_id = menu.get("restaurant_id")
+            if restaurant_id >= 235 and restaurant_id <= 249:
```

Comment:
```suggestion
            // TODO: 축제 기간 (~5/15) 종료 후 삭제
            if restaurant_id >= 235 and restaurant_id <= 249:
```
날짜가 이게 맞나..?
아무튼 이게 축제 관련 로직이고, 해당 기간 이후 삭제해야 한다고 주석 간단히 달아주세요~ 235, 249라는 수가 무슨 의미인지 파악하기 힘들 수 있어요

Reply:
그리고 이게 dev랑 prod랑 `restaurant_id`가 달라서..
어차피 간단한 임시 로직이라 dev에서 안 돌아도 상관없을 수 있지만 인지는 하고 계시면 좋을 것 같아요.

Reply:
넵 주석 달고 수정하겠습니다.
그리고 issue에도 언급은 했지만, restaurant_id가 달라서 prod에서만 돌아간다는 점은 인지하고 있습니다!
다만, dev는 일주일에 한 번만 크롤러가 돌아가기도 하고, 실제 배포가 된다면 의미가 없다고 생각해서 저렇게 구현했습니다.